src/main/java/org/skife/jdbi/v2/ClasspathStatementLocator.java

```diff
@@ -203,4 +188,11 @@ private static String left(String str, int len)
         }
         return str.substring(0, len);
     }
+
+    private static final SqlScriptParser SQL_SCRIPT_PARSER = new SqlScriptParser(new SqlScriptParser.TokenHandler() {
```

Comment:
We don't use generated `org.antlr.runtime.Parser` class. It's just a simple immutable class, which delegates events from a lexer to a token handler. A new lexer instance is created every time for a new character stream.